egs/libri_css/s5_css/run.sh

```diff
@@ -0,0 +1,243 @@
+  done
+fi
+
+#######################################################################
+# Feature extraction for the dev and eval data
+#######################################################################
+if [ $stage -le 2 ]; then
+  # mfccdir should be some place with a largish disk where you
+  # want to store MFCC features.
+  mfccdir=mfcc
+  for x in ${test_sets}; do
+    utils/fix_data_dir.sh data/$x
+    nj=$(wc -l < "data/$x/wav.scp")
+    steps/make_mfcc.sh --nj $decode_nj --cmd "$train_cmd" \
+      --mfcc-config conf/mfcc_hires.conf \
+      data/$x exp/make_mfcc/$x $mfccdir
+  done
+fi
+
+#######################################################################
+# Perform diarization on the dev/eval data
+#######################################################################
+if [ $stage -le 3 ]; then
+  for datadir in ${test_sets}; do
+    ref_rttm=data/${datadir}/ref_rttm
+    steps/segmentation/convert_utt2spk_and_segments_to_rttm.py data/${datadir}/utt2spk.bak \
+      data/${datadir}/segments.bak $ref_rttm
+    diar_nj=$(wc -l < "data/$datadir/wav.scp")
+
+    [ ! -d exp/xvector_nnet_1a ] && ./local/download_diarizer.sh
+
+    nj=$(echo $((decode_nj>diar_nj ? diar_nj : decode_nj)))
+    local/diarize_css.sh --nj $nj --cmd "$train_cmd" --stage $diarizer_stage \
+      --ref-rttm $ref_rttm --post-process-rttm true \
+      exp/xvector_nnet_1a \
+      data/${datadir} \
+      exp/${datadir}_diarization
+  done
+fi
+
+#######################################################################
+# Decode diarized output using trained chain model
+#######################################################################
+if [ $stage -le 4 ]; then
+  for datadir in ${test_sets}; do
+    asr_nj=$(wc -l < "data/$datadir/wav.scp")
+    nj=$(echo $((decode_nj>asr_nj ? asr_nj : decode_nj)))
+    local/decode_diarized_css.sh --nj $nj --cmd "$decode_cmd" --stage $decode_diarize_stage \
+      --lm-suffix "_tgsmall" --acwt 1.0 --post-decode-acwt 10.0 \
+      exp/${datadir}_diarization/rttm.post data/$datadir data/lang_test_tgsmall \
+      exp/chain${nnet3_affix}/tdnn_${affix} exp/nnet3${nnet3_affix} \
+      data/${datadir}_diarized || exit 1
+  done
+fi
+
+#######################################################################
+# Score decoded dev/eval sets (only if we are not rescoring)
+#######################################################################
+if [ $stage -le 5 ] && [ ! $rnnlm_rescore ]; then
+  # please specify both dev and eval set directories so that the search parameters
+  # (insertion penalty and language model weight) will be tuned using the dev set
+  local/score_reco_diarized.sh --stage $score_stage \
+      --multistream true \
+      --dev_decodedir exp/chain${nnet3_affix}/tdnn_${affix}/decode_${dev_set}_diarized_2stage \
+      --dev_datadir ${dev_set}_diarized_hires \
+      --eval_decodedir exp/chain${nnet3_affix}/tdnn_${affix}/decode_${eval_set}_diarized_2stage \
+      --eval_datadir ${eval_set}_diarized_hires
```

Comment:
Pass `--cmd` to this command.
```suggestion
  local/score_reco_diarized.sh --cmd "$train_cmd" --stage $score_stage \
      --multistream true \
      --dev_decodedir exp/chain${nnet3_affix}/tdnn_${affix}/decode_${dev_set}_diarized_2stage \
      --dev_datadir ${dev_set}_diarized_hires \
      --eval_decodedir exp/chain${nnet3_affix}/tdnn_${affix}/decode_${eval_set}_diarized_2stage \
      --eval_datadir ${eval_set}_diarized_hires
```